packages/sanity/src/core/form/inputs/PortableText/object/modals/PopoverModal.tsx

```diff
@@ -80,42 +78,44 @@ function Content(props: PopoverEditDialogProps) {
   const [contentElement, setContentElement] = useState<HTMLDivElement | null>(null)
   const containerElement = useRef<HTMLDivElement | null>(null)
 
-  useEffect(() => {
-    // When rendered, focus on the first input element in the content
-    if (contentElement) {
-      contentElement.querySelector('input')?.focus()
-    }
-  }, [contentElement])
+  const handleFocusLockWhiteList = useCallback((element: HTMLElement) => {
```

Comment:
Language nit: we should use a term like `allowList` instead.

Reply:
> allowList

The dependency library uses this term, so it's supposed to match that.